Tensile/KernelWriterAssembly.py

```diff
@@ -10963,11 +10965,11 @@ def emitAddressSetupCode(self, kernel, ss, tmpVgpr, tmpS01, edge, beta, atomic,
 
       # calculate global load offset
       # this has to be done after calculating new coord1
-      if not kernel["BufferStore"]:
+      if not kernel["BufferStore"] and (tc == 'D' or (tc == 'C' and beta)):
         # global: in-bounds exec mask
         # global offset macro (requires 3 tmpVgpr)
         # final address = C + index*bytes
-        kStr += "GLOBAL_OFFSET_C %u" % addrVgpr
+        kStr += "GLOBAL_OFFSET_%s %u" % (tc, addrVgpr)
```

Comment:
It looks like v_mov to tmpVgpr (line 10981,10982 after change) is redundant.
Originally, these v_mov is before setting exec flag and v_add is after exec flag.
With your change, GLOBAL_OFFSET_D is already after setting exec flag.
Then, v_mov to tmpVgpr and v_add (addr += tmpVgpr) are all within exec flag.

I guess the original intention for this part is
   (1) GLOBAL_OFFSET and v_mov to tmpVgpr for all thread
   (2) vadd within exec flag (do not execute vadd for out of range part)

However, it looks like we do not need such consideration.
We can simply add the result of GLOBAL_OFFSET with C/D addr.